CHANGES.rst

```diff
@@ -6,7 +6,8 @@ Changelog
 
 Breaking changes:
 
-- *add item here*
+- Add Plone 5 compatibility
+  [laulaz]
```

Comment:
is this breaking? IMO its a feature. Or does  it break 4.x?

Reply:
@jensens OK ... changed 👍 
Indeed, it is not breaking Plone 4 ...